### PR TITLE
Update Governance module specs

### PR DESCRIPTION
**Pull request by evanorti**

### CODE REVIEW

File: docs/develop/module-specifications/spec-governance.mdx [hunk 1]
```diff
@@ -8,25 +8,34 @@ Terra's Governance module inherits from Cosmos SDK's [`gov`](https://docs.cosmos
...
+### Deposit period
 
 After a proposal is submitted, it enters the deposit period, where it must reach a total minimum deposit of 512 Luna within 7 days from the time of its submission. The deposit threshold is reached when the sum of the initial deposit (from the proposer) and the deposits from all other interested network participants meet or exceed 512 Luna.
 
 Deposits protect against unnecessary proposals and spam.
 
 Deposits get refunded if all of the following conditions are met:
 - The minimum deposit of 512 Luna is reached within the 7-day deposit period.
-- `Quorum` is met: the number of total votes is greater than 10% of all staked Luna
+- `Quorum` is met: the number of total votes is greater than 10% of all staked Luna.
```
Comment (restorenode) [inline docs/develop/module-specifications/spec-governance.mdx]: What do you think about:
"all casted votes represent more than 10% of all staked Luna."

File: docs/develop/module-specifications/spec-governance.mdx [hunk 2]
```diff
@@ -35,31 +44,51 @@ Deposits are burned under any of the following conditions:
...
-If the minimum deposit has been reached before the end of the deposit period, then the proposal goes into a one-week voting period. While the proposal is in voting, holders of staked Luna can cast votes for the proposal. The 4 voting options available are:
+The 4 voting options available are:
 
-- `Yes` - in favor
-- `No` - not in favor
-- `NoWithVeto` - veto
-- `Abstain` - does not influence vote
+- `Yes`: in favor.
+- `No`: not in favor.
+- `NoWithVeto`: not in favor, and the creator should lose their deposit.
```
Comment (restorenode) [inline docs/develop/module-specifications/spec-governance.mdx:57]: I think the comma may not be needed in this sentence.

File: docs/develop/module-specifications/spec-governance.mdx [hunk 3]
```diff
@@ -35,31 +44,51 @@ Deposits are burned under any of the following conditions:
...
+The 4 voting options available are:
 
-- `Yes` - in favor
-- `No` - not in favor
-- `NoWithVeto` - veto
-- `Abstain` - does not influence vote
+- `Yes`: in favor.
+- `No`: not in favor.
+- `NoWithVeto`: not in favor, and the creator should lose their deposit. 
+- `Abstain`: neither for or against. An abstain vote counts toward meeting quorum but does not count toward the `threshold`.
```
Comment (restorenode) [inline docs/develop/module-specifications/spec-governance.mdx:58]: "...quorum**,** but does not count toward..."

File: docs/develop/module-specifications/spec-governance.mdx [hunk 4]
```diff
@@ -301,32 +330,33 @@ The minimum deposit amount for a proposal to enter a voting period. Currently 51
...
+The minimum number of `Yes` votes needed for a proposal to pass. Currently, 50% of all votes must be `Yes`. `Abstain` votes are excluded from this tally. 
 
 ### Veto
 
 - type: `Dec`
-- default value: `0.33`
+- default value: `0.334`
 
-Minimum value of Veto votes to Total votes ratio for proposal to be vetoed.
+The minimum value of `NoWithVeto` votes needed for a proposal to be vetoed. If 33.4% or more of the total votes cast are `NoWithVeto`, the proposal fails, and the creator's deposit is not returned.
```
Comment (restorenode) [inline docs/develop/module-specifications/spec-governance.mdx:354]: comma not needed after "the proposal fails"

**Review by restorenode — APPROVED** [review]
Consider the comments and then good to merge. 🎉

File: docs/develop/module-specifications/spec-governance.mdx (hunk 1 above)
Comment (evanorti) [reply docs/develop/module-specifications/spec-governance.mdx]: Good suggestion!

File: docs/develop/module-specifications/spec-governance.mdx (hunk 4 above)
Comment (evanorti) [reply docs/develop/module-specifications/spec-governance.mdx:354]: "the creator's deposit is not returned" is an independent clause, so a comma is needed.

File: docs/develop/module-specifications/spec-governance.mdx (hunk 3 above)
Comment (evanorti) [reply docs/develop/module-specifications/spec-governance.mdx:58]: "does not count toward the `threshold`" is a dependent clause, so it does not need a comma.

File: docs/develop/module-specifications/spec-governance.mdx (hunk 2 above)
Comment (evanorti) [reply docs/develop/module-specifications/spec-governance.mdx:57]: "the creator should lose their deposit" is an independent clause, so a comma is needed.

File: docs/develop/module-specifications/spec-governance.mdx (hunk 1 above)
Comment (evanorti) [reply docs/develop/module-specifications/spec-governance.mdx]: Implementing.

**Pull request merged**